### DISCUSSION TITLE
This Edge Function has crashed.

### DISCUSSION THREAD

**Discussion by hrutik7** [vercel/community#5131] [Help] [answered]
### Summary

when i am trying to deploy my next js project (turborepo) over the excel after build is succesfull i am getting this error
![image](https://github.com/vercel/community/assets/62372737/27ebf11f-1486-4ed5-b78c-cd1ae4d6a93c)
repo : https://github.com/hrutik7/emma

### Steps to Reproduce

import repo in vercel for deployement . after succesful build i am gettin gthis error

**Comment by hrutik7** [comment, marked as answer]
while deployement provide the env variables

**Comment by alliance1team** [comment]
hi i get the same error when i added clerk auth and i have added the env variabes in vercel but no change
